### PR TITLE
Adding connection statistics graph docs

### PR DESCRIPTION
**Pull request by josmoran-MSFT**
Add Connection Quota API (resource) and a property, ingestedItemsCount, in externalConnection entity.

### CODE REVIEW

**Comment by Arvind-Ravi** [comment]
@josmoran-MSFT - Does this need to reviewed by Shiva too. In which case you should add him as a reviewer too.

**Comment by josmoran-MSFT** [comment]
> @josmoran-MSFT - Does this need to be reviewed by Shiva too. In which case you should add him as a reviewer too.

@Arvind-Ravi, Yes he should review it as well. Thanks for the reminder, I will add him as a reviewer as well.

**Review by mecampos — APPROVED** [review]

File: api-reference/beta/resources/externalconnectors-connectionQuota.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+## Methods
+|Method|Return type|Description|
+|:---|:---|:---|
+| [Get quota](../api/externalconnectors-connectionQuota-get.md) |[connectionQuota](../resources/externalconnectors-connectionQuota.md)| calculated information on the connection's quota utilization. |
+
+## Properties
+|Property|Type|Description|
+|:---|:---|:---|
+| itemsRemaining | Int64 | Returns minimum of **items remaining in connection** and **remaining items at tenant-level** which is represented by this equation: `min(Max capacity in connection – Number of items in connection, Tenant Quota – Number of items indexed in all connections)`. If connection is non-monetized (preview connector or preview content experience), then returns the number of remaining items in connection. |
```
Comment (JarbasHorst) [inline api-reference/beta/resources/externalconnectors-connectionQuota.md:24]: Please, 
- add "the" before "minimum"
- add "the" before "connection" => remaining in the connection. 
- add "the" after "If"
- add "it" after "then"
- add "the" before the last appearance of "connection"

File: changelog/Microsoft.Search.json [hunk 2]
```diff
@@ -799,20 +799,37 @@
     {
       "ChangeList": [
         {
-          "Id": "b0cf779b-1461-4e8e-afd7-2896bf5bfd29",
```
Comment (JarbasHorst) [inline changelog/Microsoft.Search.json:802]: It looks like you are adding your changes to an existing changelog record. Please, don't override existing changelog records and add your changes to a new changelog record.

File: api-reference/beta/api/externalconnectors-connectionQuota-get.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
+---
+title: "Get connectionQuota"
```
Comment (JarbasHorst) [inline api-reference/beta/api/externalconnectors-connectionQuota-get.md:2]: Please, include an update to the toc.yml file in the beta folder, to surface this API in the TOC.

File: api-reference/beta/api/externalconnectors-connectionQuota-get.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+
+
+# [HTTP](#tab/http)
+<!-- {
+  "blockType": "request",
+  "name": "get_connection"
+}-->
+
+```msgraph-interactive
+GET https://graph.microsoft.com/beta/external/connections/contosohr/quota
```
[the remark this answers is not in the document]
Comment (josmoran-MSFT) [reply api-reference/beta/api/externalconnectors-connectionQuota-get.md:69]: @JarbasHorst - No this is an example ID

[the remark this answers is not in the document]
Comment (JarbasHorst) [reply api-reference/beta/api/externalconnectors-connectionQuota-get.md:69]: Ideally, please replace "contosohr" with a "real" example that demonstrates the correct usage of this API request.

File: api-reference/beta/resources/enums-externalconnectors.md [hunk 5]
```diff
@@ -53,6 +53,12 @@ Namespace: microsoft.graph.externalConnectors
 | limitExceeded
 | unknownFutureValue
 
+### connectionQuota values
```
[the remark this answers is not in the document]
Comment (josmoran-MSFT) [reply api-reference/beta/resources/enums-externalconnectors.md:56]: @JarbasHorst, yes connectionQuota referring to the resource topic. But it is not an enum.

File: api-reference/beta/api/externalconnectors-connectionQuota-get.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (josmoran-MSFT) [reply api-reference/beta/api/externalconnectors-connectionQuota-get.md:69]: Oh sorry, then if you are referring to whether if it is the correct usage, then yes it does represent an ID that can be used in this request.

File: changelog/Microsoft.Search.json (hunk 2 above)
Comment (josmoran-MSFT) [reply changelog/Microsoft.Search.json:802]: @JarbasHorst, thanks for all of your feedback!

Although on I don't understand. If its related to Microsoft Search docs I cannot include the change logs to this file?

I see others replicating this ideas where they would constantly override an existing change log.

Comment (josmoran-MSFT) [reply changelog/Microsoft.Search.json:802]: image

Comment (josmoran-MSFT) [reply changelog/Microsoft.Search.json:802]: For example ^

File: api-reference/beta/api/externalconnectors-connectionQuota-get.md [hunk 6]
```diff
@@ -0,0 +1,117 @@
+---
```
Comment (JarbasHorst) [inline api-reference/beta/api/externalconnectors-connectionQuota-get.md:1]: For consistency, this filename should be in lower case format. Please, rename this file to => **externalconnectors-connectionquota-get.md**.

File: api-reference/beta/resources/externalconnectors-connectionQuota.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
+---
```
Comment (JarbasHorst) [inline api-reference/beta/resources/externalconnectors-connectionQuota.md:1]: For consistency, this filename should be in lower case format. Please, rename this file to =>  **externalconnectors-connectionquota.md**

File: api-reference/beta/toc.yml [hunk 8]
```diff
@@ -13798,6 +13798,10 @@ items:
             href: api/externalconnectors-externalgroup-post-members.md
           - name: Delete external group member
             href: api/externalconnectors-externalgroupmember-delete.md
+        - name: Connnection Quota
```
Comment (JarbasHorst) [inline api-reference/beta/toc.yml:13801]: For consistency, please replace "Quota" with "quota". Also, please replace "Co**n**nnection" with "Connection".

File: api-reference/beta/toc.yml [hunk 9]
```diff
@@ -13798,6 +13798,10 @@ items:
...
+          items:
+          - name: Get Connection Quota
```
Comment (JarbasHorst) [inline api-reference/beta/toc.yml:13803]: For consistency, please replace "Connection" with "connection" and  "Quota" with "quota".

File: api-reference/beta/toc.yml [hunk 10]
```diff
@@ -13798,6 +13798,10 @@ items:
...
+            href: api/externalconnectors-connectionQuota.md
```
Comment (JarbasHorst) [inline api-reference/beta/toc.yml:13804]: Please, replace this link with "api/externalconnectors-connectionquota-get.md".

File: api-reference/beta/toc.yml [hunk 11]
```diff
@@ -13798,6 +13798,10 @@ items:
             href: api/externalconnectors-externalgroup-post-members.md
           - name: Delete external group member
             href: api/externalconnectors-externalgroupmember-delete.md
+        - name: Connnection Quota
+          items:
```
Comment (JarbasHorst) [inline api-reference/beta/toc.yml:13802]: The first link should refer to the resource topic. Please, add a reference to the resource topic to the TOC. Suggested revision:

```yaml
- name: Connection quota
  items:
    - name: Connection quota
    - href: resources/externalconnectors-connectionquota.md 
    - name: Get connection quota
    - href: api/externalconnectors-connectionquota-get.md
```

File: api-reference/beta/resources/externalconnectors-connectionQuota.md [hunk 12]
```diff
@@ -0,0 +1,46 @@
...
+
+Namespace: microsoft.graph.externalConnectors
+
+The [connection](externalconnectors-externalconnection.md) quota contains calculated information on the connection's quota utilization. It returns the allowed number of items that you can ingest into a connection by considering items ingested for the connection against the tenant-level quota for Microsoft Graph connectors.
+
+## Methods
+
+|Method|Return type|Description|
+|:---|:---|:---|
+| [Get connectionQuota](../api/externalconnectors-connectionQuota-get.md) |[connectionQuota](../resources/externalconnectors-connectionQuota.md)| Retrieve the properties and relationships of a **connectionQuota**. |
```
Comment (JarbasHorst) [inline api-reference/beta/resources/externalconnectors-connectionQuota.md:19]: For consistency, **both** appearances of "externalconnectors-**connectionQuota**" should be in lower case format => externalconnectors-**connectionquota**

File: api-reference/beta/resources/externalconnectors-connectionQuota.md [hunk 13]
```diff
@@ -0,0 +1,46 @@
...
+  "blockType": "resource",
+  "keyProperty": "id",
+  "@odata.type": "microsoft.graph.externalConnectors.connectionQuota",
+  "openType": false
+}
+-->
+
+``` json
+{
+  "itemsRemaining": Int64
```
Comment (JarbasHorst) [inline api-reference/beta/resources/externalconnectors-connectionQuota.md:44]: For conformity, please add quotes to Int64 =>  "itemsRemaining": "Int64"

File: api-reference/beta/api/externalconnectors-connectionQuota-get.md [hunk 14]
```diff
@@ -0,0 +1,117 @@
...
+|:--------------|:--------------------------|
+| Authorization | Bearer {token}. Required. |
+
+## Request body
+
+Do not supply a request body for this method.
+
+## Response
+
+If successful, this method returns a `200 OK` response code and the requested **[externalConnection](../resources/externalconnectors-externalconnection.md)** object in the response body.
```
Comment (JarbasHorst) [inline api-reference/beta/api/externalconnectors-connectionQuota-get.md:52]: For consistency, please replace "the requested" with "an". Also, please remove the bold formatting from externalConnection.

File: api-reference/beta/api/externalconnectors-connectionQuota-get.md [hunk 15]
```diff
@@ -0,0 +1,117 @@
...
+  "truncated": true,
+  "@odata.type": "microsoft.graph.externalConnectors.connectionQuota"
+} -->
+
+```http
+HTTP/1.1 200 OK
+Content-type: application/json
+
+{
+  "itemsRemaining": Int64
```
Comment (JarbasHorst) [inline api-reference/beta/api/externalconnectors-connectionQuota-get.md:105]: For consistency, the Response body should not contain the JSON representation of the object. In this case, please replace "Int64" with 700000, which I think was the previous value here, right?

File: api-reference/beta/api/externalconnectors-externalconnection-get.md [hunk 16]
```diff
@@ -49,7 +49,7 @@ Do not supply a request body for this method.
 
 ## Response
 
-If successful, this method returns a `200 OK` response code and the requested [externalConnection](../resources/externalconnectors-externalconnection.md) object in the response body.
+If successful, this method returns a `200 OK` response code and the requested **[externalConnection](../resources/externalconnectors-externalconnection.md)** object in the response body.
```
Comment (JarbasHorst) [inline api-reference/beta/api/externalconnectors-externalconnection-get.md:52]: For consistency, please replace "the requested" with "an". Also, please remove the bold formatting from externalConnection.

File: api-reference/beta/api/externalconnectors-externalconnection-get.md [hunk 17]
```diff
@@ -110,6 +110,7 @@ Content-type: application/json
 
 {
   "id": "contosohr",
+  "ingestedItemsCount": Int64,
```
Comment (JarbasHorst) [inline api-reference/beta/api/externalconnectors-externalconnection-get.md:113]: For consistency, the Response body should not contain the JSON representation of the object. In this case, please replace "Int64" with 700000, which I think was the previous value here, right?

File: api-reference/beta/api/externalconnectors-externalconnection-list.md [hunk 18]
```diff
@@ -113,6 +113,7 @@ Content-type: application/json
       "name": "Contoso HR",
       "description": "Connection to index Contoso HR system",
       "state": "ready",
+      "ingestedItemsCount": Int64,
```
Comment (JarbasHorst) [inline api-reference/beta/api/externalconnectors-externalconnection-list.md:116]: For consistency, the Response body should not contain the JSON representation of the object. In this case, please replace "Int64" with 700000, which I think was the previous value here, right? 

Please apply this same change to line 128.

File: api-reference/beta/api/externalconnectors-externalconnection-list.md [hunk 19]
```diff
@@ -49,7 +49,7 @@ Do not supply a request body for this method.
 
 ## Response
 
-If successful, this method returns a `200 OK` response code and a collection of [externalConnection](../resources/externalconnectors-externalconnection.md) objects in the response body.
+If successful, this method returns a `200 OK` response code and a collection of **[externalConnection](../resources/externalconnectors-externalconnection.md)** objects in the response body.
```
Comment (JarbasHorst) [inline api-reference/beta/api/externalconnectors-externalconnection-list.md:52]: For consistency, please remove the bold formatting from externalConnection.

File: api-reference/beta/resources/externalconnectors-externalconnection.md [hunk 20]
```diff
@@ -19,14 +19,15 @@ A logical container to add content from an external source into Microsoft Graph.
...
+| [Delete externalConnection](../api/externalconnectors-externalconnection-delete.md) | None                                          | Delete a **externalConnection** object. |
 | [Create schema](../api/externalconnectors-externalconnection-post-schema.md)        | None *or* [schema](externalconnectors-schema.md)                 | Register connection schema. |
 | [Get operation](../api/externalconnectors-connectionoperation-get.md)               | [connectionOperation](externalconnectors-connectionoperation.md) | Get the status of an asynchronous request to create the connection schema. |
 | [Create externalItem](../api/externalconnectors-externalconnection-put-items.md)    | [externalItem](externalconnectors-externalitem.md)               | Create a new externalItem by posting to the items collection. |
+|[Get quota](../api/externalconnectors-connectionQuota-get.md)|[connectionQuota](../resources/externalconnectors-connectionQuota.md)|Calculate the quota utilization of an **externalConnection** object.|
```
Comment (JarbasHorst) [inline api-reference/beta/resources/externalconnectors-externalconnection.md:30]: 1) For consistency, please replace "Get quota" with "Get connectionQuota".

2) For consistency, "externalconnectors-**connectionQuota**-get.md" should be in lower case format => externalconnectors-**connectionquota**-get.md
3) Also, the description provided here should be the same as the description provided in the API topic. So, "Calculate the quota utilization of an **externalConnection** object." should be replaced with "Retrieve the properties and relationships of a **connectionQuota**.". If the sentence "Calculate the quota utilization of an **externalConnection** object." is relevant, I recommend you to add it as part of the description for the Get connectionQuota API topic.

File: api-reference/beta/resources/externalconnectors-externalconnection.md [hunk 21]
```diff
@@ -36,6 +37,7 @@ A logical container to add content from an external source into Microsoft Graph.
 | connectorId   |String                             | The Teams App ID. Optional.|
 | description   | String                            | Description of the connection displayed in the Microsoft 365 admin center. Optional. |
 | id            | String                            | Developer-provided unique ID of the connection within the Azure Active Directory tenant. Must be between 3 and 32 characters in length. Must only contain alphanumeric characters. Cannot begin with `Microsoft` or be one of the following values: `None`, `Directory`, `Exchange`, `ExchangeArchive`, `LinkedIn`, `Mailbox`, `OneDriveBusiness`, `SharePoint`, `Teams`, `Yammer`, `Connectors`, `TaskFabric`, `PowerBI`, `Assistant`, `TopicEngine`, `MSFT_All_Connectors`. Required. |
+| ingestedItemsCount           | Integer                            |  The number of items ingested into a connection. This value is refreshed every 15 minutes. If the connection state is `draft`, then **ingestedItemsCount** will be `null`. |
```
Comment (JarbasHorst) [inline api-reference/beta/resources/externalconnectors-externalconnection.md:40]: For consistency, please replace "Integer" with "Int64"

File: api-reference/beta/api/externalconnectors-connectionQuota-get.md [hunk 22]
```diff
@@ -0,0 +1,97 @@
...
+  "truncated": true,
+  "@odata.type": "microsoft.graph.externalConnectors.connectionQuota"
+} -->
+
+```http
+HTTP/1.1 200 OK
+Content-type: application/json
+
+{
+  "itemsRemaining": 7000
```
Comment (JarbasHorst) [inline api-reference/beta/api/externalconnectors-connectionQuota-get.md:85]: In all your other examples, the value is 700000. Not sure if this is relevant. If yes, please replace 7000 with 700000.

File: api-reference/beta/api/externalconnectors-externalconnection-get.md [hunk 23]
```diff
@@ -49,17 +49,14 @@ Do not supply a request body for this method.
...
 ## Response
 
-If successful, this method returns a `200 OK` response code and the requested [externalConnection](../resources/externalconnectors-externalconnection.md) object in the response body.
+If successful, this method returns a `200 OK` response code and an [externalConnection](../resources/externalconnectors-externalconnection.md) object in the response body.
 
 ## Examples
 
 ### Request
 
 The following is an example of the request.
```
Comment (JarbasHorst) [inline api-reference/beta/api/externalconnectors-externalconnection-get.md:58]: For consistency, please replace "the" before "request" with "a" (line 58). Suggested revision:

- The following is an example of a request.

File: api-reference/beta/api/externalconnectors-externalconnection-list.md [hunk 24]
```diff
@@ -57,7 +57,6 @@ If successful, this method returns a `200 OK` response code and a collection of
 
 The following is an example of the request.
```
Comment (JarbasHorst) [inline api-reference/beta/api/externalconnectors-externalconnection-list.md:58]: For consistency, please replace "the" before "request" with "a" (line 58). Suggested revision:

- The following is an example of a request.

File: api-reference/beta/resources/externalconnectors-connectionQuota.md [hunk 25]
```diff
@@ -0,0 +1,46 @@
...
+
+|Method|Return type|Description|
+|:---|:---|:---|
+| [Get connectionQuota](../api/externalconnectors-connectionquota-get.md) |[connectionQuota](../resources/externalconnectors-connectionquota.md)| Retrieve the properties and relationships of a **connectionQuota**. |
+
+## Properties
+
+|Property|Type|Description|
+|:---|:---|:---|
+| itemsRemaining | Int64 | Returns the minimum of **items remaining in the connection** and **remaining items at tenant-level** which is represented by this equation: `min(Max capacity in the connection – Number of items in connection, Tenant Quota – Number of items indexed in all connections)`. If the connection is non-monetized (preview connector or preview content experience), then it returns the number of remaining items in the connection. |
```
Comment (JarbasHorst) [inline api-reference/beta/resources/externalconnectors-connectionQuota.md:25]: For clarity, I recommend you to rephrase this description. Suggested revision:

- Returns the minimum number in a range that contains the following data: \*items remaining in the connection\* and \*remaining items at tenant-level\*. The following equation represents the formula used to calculate the minimum number: \`min(max capacity in the connection – number of items in the connection, tenant quota – number of items indexed in all connections)\`. If the connection is nonmonetized (preview connector or preview content experience), then it returns the number of remaining items in the connection.

File: changelog/Microsoft.Search.json [hunk 26]
```diff
@@ -813,6 +813,41 @@
...
-    }    
+    },
+    {
+      "ChangeList": [
+        {
+          "Id": "384321d5-40b4-4496-9636-c7bf9150599b",
+          "ApiChange": "Resource",
+          "ChangedApiName": "connectionQuota",
+          "ChangeType": "Addition",
+          "Description": "Added the [connectionQuota](https://docs.microsoft.com/en-us/graph/api/resources/connectionQuota?view=graph-rest-beta) resource",
```
Comment (JarbasHorst) [inline changelog/Microsoft.Search.json:824]: Please, add "and a supported method" after "resource" and a period at the end of the sentence. Also, please replace connectionQuota in the URL with "externalconnectors-connectionquota". Suggested revision:

- Added the \[connectionQuota\](https://docs.microsoft.com/en-us/graph/api/resources/externalconnectors-connectionquota?view=graph-rest-beta) resource and a supported method.

File: changelog/Microsoft.Search.json [hunk 27]
```diff
@@ -813,6 +813,41 @@
...
+          "ChangeType": "Addition",
+          "Description": "Added the [quota](https://docs.microsoft.com/en-us/graph/api/resources/externalConnection?view=graph-rest-beta) resource",
+          "Target": "externalConnection"
+        }
+      ],
+      "Id": "384321d5-40b4-4496-9636-c7bf9150599b",
+      "Cloud": "Prod",
+      "Version": "beta",
+      "CreatedDateTime": "2021-12-08T23:52:35.8265267Z",
+      "WorkloadArea": "Substrate Connectors",
```
Comment (JarbasHorst) [inline changelog/Microsoft.Search.json:848]: The correct workload area for your PR is "Search". In this case, please replace "Substrate Connectors" with "Search".

File: changelog/Microsoft.Search.json [hunk 28]
```diff
@@ -813,6 +813,41 @@
...
+          "ChangeType": "Addition",
+          "Description": "Added the [connectionQuota](https://docs.microsoft.com/en-us/graph/api/resources/connectionQuota?view=graph-rest-beta) resource",
+          "Target": "connectionQuota"
+        },
+        {
+          "Id": "384321d5-40b4-4496-9636-c7bf9150599b",
+          "ApiChange": "Property",
+          "ChangedApiName": "ingestedItemsCount",
+          "ChangeType": "Addition",
+          "Description": "Added the [ingestedItemsCount](https://docs.microsoft.com/en-us/graph/api/resources/externalConnection?view=graph-rest-beta) resource",
```
Comment (JarbasHorst) [inline changelog/Microsoft.Search.json:832]: Please, rephrase this description. Suggested revision:

- Added the \*\*\ingestedItemsCount\*\* property to the \[externalConnection\](https://docs.microsoft.com/en-us/graph/api/resources/externalconnectors-externalconnection?view=graph-rest-beta) resource.

File: changelog/Microsoft.Search.json [hunk 29]
```diff
@@ -813,6 +813,41 @@
...
+          "Target": "externalConnection"
+        },
+        {
+          "Id": "384321d5-40b4-4496-9636-c7bf9150599b",
+          "ApiChange": "Resource",
```
Comment (JarbasHorst) [inline changelog/Microsoft.Search.json:837]: Please, replace "Resource" with "Relationship".

File: changelog/Microsoft.Search.json [hunk 30]
```diff
@@ -813,6 +813,41 @@
...
+          "ChangedApiName": "quota",
+          "ChangeType": "Addition",
+          "Description": "Added the [quota](https://docs.microsoft.com/en-us/graph/api/resources/externalConnection?view=graph-rest-beta) resource",
```
Comment (JarbasHorst) [inline changelog/Microsoft.Search.json:840]: Please, rephrase this description. Suggested revision:

- Added the **quota** relationship to the \[externalConnection\](https://docs.microsoft.com/en-us/graph/api/resources/externalconnectors-externalconnection?view=graph-rest-beta) resource.

File: api-reference/beta/toc.yml [hunk 31]
```diff
@@ -13929,6 +13929,12 @@ items:
             href: api/externalconnectors-externalgroup-post-members.md
           - name: Delete
             href: api/externalconnectors-externalgroupmember-delete.md
+        - name: Connection quota
+          items:
+          - name: Connection quota
+          - href: resources/externalconnectors-connectionquota.md
```
Comment (JarbasHorst) [inline api-reference/beta/toc.yml:13935]: For conformity, please remove the dash before href. Same for line 13937. Suggested revision:

```
- name: Connection quota
  href: resources/externalconnectors-connectionquota.md
- name: Get connection quota
  href: api/externalconnectors-connectionquota-get.md
```

File: api-reference/beta/resources/externalconnectors-externalconnection.md [hunk 32]
```diff
@@ -44,10 +46,11 @@ A logical container to add content from an external source into Microsoft Graph.
 
 | Relationship | Type                                                     | Description |
 |:-------------|:---------------------------------------------------------|:---|
```
Comment (JarbasHorst) [inline api-reference/beta/resources/externalconnectors-externalconnection.md:48]: For clarity, please provide a more descriptive description for the items in this list. For example, 

| Relationship | Type                                                     | Description |
|:-------------|:---------------------------------------------------------|:---|
| groups       | [microsoft.graph.externalConnectors.externalGroup](externalconnectors-externalgroup.md) collection             | A collection of  non-Azure Active Directory groups. Read-only. Nullable. |
| items        | [microsoft.graph.externalConnectors.externalItem](externalconnectors-externalitem.md) collection               | A list of items added to an \*\*externalConnection\*\* in Microsoft Graph. Read-only. Nullable. |
| and so on... | | |

**Comment by JarbasHorst** [comment]
@josmoran-MSFT, please fix the remaining issues below so that we can merge and publish this PR:

- Resolve the conflicts
- For clarity, please provide a more informative description for the items in this list. Corresponding resource topic: api-reference/beta/resources/externalconnectors-externalconnection.md. For example, 

| Relationship | Type                                                     | Description |
|:-------------|:---------------------------------------------------------|:---|
| groups       | [microsoft.graph.externalConnectors.externalGroup](externalconnectors-externalgroup.md) collection             | A collection of  non-Azure Active Directory groups. Read-only. Nullable. |
| items        | [microsoft.graph.externalConnectors.externalItem](externalconnectors-externalitem.md) collection               | A list of items added to an \*\*externalConnection\*\* in Microsoft Graph. Read-only. Nullable. |
| and so on... | | |

**Comment by JarbasHorst** [comment]
@Lauragra this PR is ready for final review. Please, merge PR [16430](https://github.com/microsoftgraph/microsoft-graph-docs/pull/16430) into this branch to include the resolved merge conflicts.

**Pull request merged**